__tests__/app.test.js

```diff
@@ -78,6 +78,155 @@ describe("/api/reviews", () => {
+          
+          });
+      });
+
+      test("accepts order query", () => {
+        return request(app)
+        .get("/api/reviews?order=asc")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.reviews).toBeInstanceOf(Array);
+          expect(body.reviews).toHaveLength(13);
+
+          body.reviews.forEach((review) => {
+            expect(review).toMatchObject({
+              owner: expect.any(String),
+              title: expect.any(String),
+              review_id: expect.any(Number),
+              category: expect.any(String),
+              review_img_url: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              designer: expect.any(String),
+              comment_count: expect.any(Number),
+            });
+          });
+
+          expect(body.reviews).toBeSortedBy("created_at", { ascending: true });
+        
+        });
+      });
+
+      test("all queries work together", () => {
+        return request(app)
+        .get("/api/reviews?order=asc&sort_by=title&category=social deduction")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.reviews).toBeInstanceOf(Array);
+          expect(body.reviews).toHaveLength(11);
+
+          body.reviews.forEach((review) => {
+            expect(review).toMatchObject({
+              owner: expect.any(String),
+              title: expect.any(String),
+              review_id: expect.any(Number),
+              category: expect.any(String),
+              review_img_url: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              designer: expect.any(String),
+              comment_count: expect.any(Number),
+            });
+          });
+
+          expect(body.reviews).toBeSortedBy("title", { ascending: true });
+        
+        });
+      });
+
+      test('if invalid sort input given, return error', () => {
+        return request(app)
+        .get("/api/reviews?sort_by=hello")
+        .expect(400)
+        .then(({body}) =>{
+          expect(body.msg).toBe('Invalid sort query')
+        })
+      });
+
+      test('if invalid order input given, return error', () => {
+        return request(app)
+        .get("/api/reviews?order=hello")
+        .expect(400)
+        .then(({body}) =>{
+          expect(body.msg).toBe('Invalid order query')
+        })
+      });
+
+      test('if invalid order input given, return error', () => {
+        return request(app)
+        .get("/api/reviews?category=hello")
+        .expect(400)
+        .then(({body}) =>{
+          expect(body.msg).toBe('Given category does not exist')
+        })
+      });
+
+      test('if invalid query is given, return error', () => {
+        return request(app)
+        .get("/api/reviews?name=hello")
+        .expect(400)
+        .then(({body}) =>{
+          expect(body.msg).toBe('Invalid query')
+        })
+      });
```

Comment:
Nothing to do with the delete, BUT another test you could add in regards to your query tests:
- valid category that doesn't have any reviews -> should respond with an empty array and give a status of 200